### PR TITLE
Add TSC meeting notes for 11-02-2020

### PR DESCRIPTION
**Pull request by michdolan**
Signed-off-by: Michael Dolan <michdolan@gmail.com>

### CODE REVIEW

File: ASWF/meetings/tsc/2020-11-02.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+    - Kevin: Can do dynamic functionality right now with search paths, etc. We 
+      abuse that for CDL selection, an explicit choice.
+    - Doug: Thanks for the feedback. We have response for all of these. In 
+      terms of written response, would you like answers in this doc, or in 
+      separate doc?
+    - J: Separate doc is preferred.
+    - Michael: Will be good to share these topics and conversation with wider 
+      community too, for additional input. We can continue this discussion 
+      after we all have time to think through the concerns raised.
+
```
Comment (doug-walker) [inline ASWF/meetings/tsc/2020-11-02.md:197]: Please add:

    - *Update:* After the meeting, Doug provided a written response to the ILM questions:
<https://docs.google.com/document/d/1HeA9fJvrxAVoUQ3M1p8K3gAgd7cLkHUmChCoPfmyZvQ/edit?usp=sharing>

File: ASWF/meetings/tsc/2020-11-02.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+      numbers, there's an extra bit to account for.
+    - Kevin: If I got a config file with parameters that would be helpful. At 
+      moment we get CDLs and stuff that should match, but they don't sometimes.
+    - J: The variability of these controls is something to help facilitate 
+      re-baking LUTs. There's value in this, but our concern is this will 
+      create dynamic version of this OCIO experience that is not intended. Can 
+      forsee a future that used builtin grading tool that uses these transforms 
+      and export that to handoff to studios, but would be a nightmare. 
+    - Mark B: Or combining grading transforms in row. What will that mean?
+    - Kevin: Nothing stopping onset grading from baking out CTL files, so not
```
Comment (doug-walker) [inline ASWF/meetings/tsc/2020-11-02.md:119]: CTL --> CDL

**Review by doug-walker — CHANGES_REQUESTED** [review]

File: ASWF/meetings/tsc/2020-11-02.md (hunk 2 above)
Comment (michdolan) [reply ASWF/meetings/tsc/2020-11-02.md:119]: @KevinJW did you mean CDL here, or is CTL correct? I know we discussed both possibilities.

File: ASWF/meetings/tsc/2020-11-02.md (hunk 1 above)
Comment (michdolan) [reply ASWF/meetings/tsc/2020-11-02.md:200]: Thanks @doug-walker . Added.

File: ASWF/meetings/tsc/2020-11-02.md (hunk 2 above)
Comment (KevinJW) [reply ASWF/meetings/tsc/2020-11-02.md:119]: CTL might work if a bit painful

CLF/CTF (not CDL), I believe I was suggesting that as an extension of CDLs there is no reason we could not have a more expressive way of passing on the on-set/dailies data (other than industry agreement !)

In this case with dynamic parameters on set, are conceptually just the same as CDL parameters on set in terms of impact, neither is sufficient you need to pass on the whole transformation chain, which could be baked to CLF/CTF. The fact that it is dynamic "on set" or in dailies is nothing new.

Comment (michdolan) [reply ASWF/meetings/tsc/2020-11-02.md]: Fixed.

**Review by doug-walker — APPROVED** [review]

**Review by hodoulp — APPROVED** [review]

**Pull request merged**